api/src/main/resources/client_api.yaml

```diff
@@ -74,6 +74,57 @@ paths:
           schema:
             $ref: "#/definitions/MaterializeCohortResponse"
 
+  /v1/workspaces/{workspaceNamespace}/{workspaceId}/concepts:
+      get:
+        security: []
```

Comment:
This shouldn't be used in combination with a workspace path. You should need access to a workspace in order to call any method on it, even to know whether or not it exists. Should require auth or not make this workspace-centric.

Reply:
Auth required now, good catch.